source/_integrations/bring.markdown

```
@@ -89,3 +117,81 @@
       message: urgent_message
       item: Cilantro
 ```
+
+## Automations
+
+Get started with these automation examples for **Bring!**, each featuring ready-to-use blueprints!
+
+#### Grocery shopping reminder 🛒
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)